tugaphone/tokenizer.py

```
@@ -2109,6 +3127,9 @@ def demonstrate_transcription(text: str, dialect: DialectInventory = None):
     if dialect is None:
         dialect = EuropeanPortuguese()
 
+    from tugaphone.pos import TugaTagger
+    pos = TugaTagger(engine="brill")
```

Ruff (0.14.14)

[error] Local variable `pos` is assigned to but never used

Remove assignment to unused variable `pos`

(F841)